lib/ansible/modules/files/find.py

```diff
@@ -44,7 +44,7 @@
     excludes:
         description:
             - One or more (shell or regex) patterns, which type is controlled by C(use_regex) option.
-            - Items matching an C(excludes) pattern are culled from C(patterns) matches.
+            - Files whose basenames match an C(excludes) pattern are culled from C(patterns) matches.
```

Comment:
don't change Items to Files, it is not always the case that the base name is a file

Reply:
A few lines above, it says:

> The patterns restrict the list of files to be returned to those whose basenames match

Is my change not consistent with that description?

Reply:
consistent .. but still not right